dotnet-docs/azure-marketplace-integration.md

```diff
@@ -0,0 +1,99 @@
+
+1. Configure the __Virtual Machine__ section. The screenshot below shows the recommended _Ubuntu OS Version_, _CPU Architecture_, and _VM size_:
+
+	![Configuring the Virtual Machine section of the Report Server .NET resource on Azure Marketplace.](../images/rs-net-images/rs-net-azure-marketplace-settings-virtual-machine.png)
+
+	You may use an existing IP Address, or create a new one with the button at the bottom of the _Public IP Address for the VM_ setting.
+
+	If you choose an existing IP Address, before starting to deploy the Report Server in Azure, you have to buy a domain name, create a DNS Zone, and a Public IP address. Make the _A_ record of the DNS zone to have the value of your public IP address. The DNS Zone and the Public IP address must be in a separate Resource group, because deploying the Report Server needs to happen in an empty resource group. Then, you can pick the Public IP address to which you have connected the Domain name. You may use this domain name in the _HTTPS_ section to choose a Free Let's Encrypt HTTPS Certificate.
+
+	>important The __DNS Prefix for the public IP Address__ will be concatenated with the string below it to form the Report Server URL, where you will be able to access the Report Server Manager.
+
+1. Add the __License__ token for your Telerik Report Server product - see how to set up the [Telerik Report Server License]({%slug license-key%}):
+
+	![Configuring the License section of the Report Server .NET resource on Azure Marketplace.](../images/rs-net-images/rs-net-azure-marketplace-settings-license.png)
+
+1. Configure the __Administrator__ settings:
+
+	![Configuring the Administrator section of the Report Server .NET resource on Azure Marketplace.](../images/rs-net-images/rs-net-azure-marketplace-settings-administrator.png)
+
+1. The __Mail__ settings are optional. You may configure the SMTP in the Report Server configuration after the deployment.
+
+	![Configuring the optional Mail section of the Report Server .NET resource on Azure Marketplace.](../images/rs-net-images/rs-net-azure-marketplace-settings-mail.png)
+
+1. The __Storage__ section lets you provide a connection string to an external MSSQL Database. Otherwise, the default storage will be an MSSQL Express database deployed in a Docker container in the VM.
+
+ 	![Configuring the optional Storage section of the Report Server .NET resource on Azure Marketplace.](../images/rs-net-images/rs-net-azure-marketplace-settings-storage.png)
+
+1. In __HTTPS__, you may choose a Free Let's Encrypt HTTPS Certificate.
+
+	If you want to use the Free Let's Encrypt HTTPS Certificate, before starting to deploy the Report Server in Azure, you have to buy a domain name, create a DNS Zone, and a Public IP address as explained in the _Virtual Machine_ section. Enter here your domain name created in the _Virtual Machine_ section.
```

Comment:
Enter here your domain name created in the _Virtual Machine_ section - it will be the domain name they bought themselves and they never used it in the setup process before this step.

Reply:
done